### ISSUE TITLE
BBCode Justify left button state doesn't have proper state

### ISSUE THREAD

**Issue by engineering-this** [ckeditor/ckeditor4#2389]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Open test:
`tests/plugins/bbcode/manual/justify`
2. Put caret in editor.
3. Press `justify left` button.
4. Go to source mode and back to wysiwig mode.
5.  Put caret in same place.

### Expected result

`Justify left` button is active.

### Actual result

`Justify left` button is inactive.

Note: this issue is valid on `major` branch.

## Other details

* Browser: Any
* OS: any
* CKEditor version: 4.11.0
* Installed CKEditor plugins: bbcode, justify

**Comment by mlewand** [comment]
I don't think that the result after 2nd should be expected. The alignment is not explicitly set, so it should not be marked as aligned. Because it depends from parent CSS. Button should only be active if the markup explicitly sets the direction.

As of the bug at the end it seems that it is caused by the fact that there is added a `div` with no alignment style.
